codice/ddf
# DDF-04447 Migrate the user-blacklist component to React

#### What does this PR do?
Migrates the user-blacklist component to React
#### Who is reviewing it? 
@gjvera 
@mojogitoverhere 
#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining
@djblue 

#### How should this be tested?
1. Ingest some data
2. Run a search
3. Hide some of the results by using the 3 dot menu on the individual results
4. View the blacklist by clicking the cog in the top right corner of intrigue and then clicking `Hidden`
5. Ensure no regressions are present
    - Test with different theme settings
#### Any background context you want to provide?
The CSS was somehow hiding the unhide button on the individual records on master, so I had to guess as to what that should look like.

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4447 

#### Screenshots

![image](https://user-images.githubusercontent.com/10997562/53422015-e834cf80-399b-11e9-9df3-6f0f6b42c067.png)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
